src/reynir_correct/checker.py

```diff
@@ -186,17 +186,18 @@ class GreynirCorrect(Greynir):
     _lock = Lock()
 
     def __init__(self, **options: Any) -> None:
-        super().__init__()
+        super().__init__(**options)
```

Comment:
Strictly speaking, the super().__init__() call should come after the options.pop() call. The superclass should not need to know or worry about options that are specific to derived classes, and indeed derived classes should be able to override option meanings.

Reply:
Done